### ISSUE TITLE
grouplist selection search errors

### ISSUE THREAD

**Issue by mariari** [stumpwm/stumpwm#965]
After 

https://github.com/stumpwm/stumpwm/commit/4ddd4600036a2c80bd3a12c08a57a67ceee24f21

`grouplist` errors with `The function COMMON-LISP:NIL is undefined.` when typing any text into the `grouplist` search prompt.

Rolling back this commit fixes the issue.

Here the debug log from sly/swank

```lisp
The function COMMON-LISP:NIL is undefined.
   [Condition of type UNDEFINED-FUNCTION]

Restarts:
 0: [CONTINUE] Retry calling NIL.
 1: [USE-VALUE] Call specified function.
 2: [RETURN-VALUE] Return specified values.
 3: [RETURN-NOTHING] Return zero values.
 4: [RETRY] Retry SLY mREPL evaluation request.
 5: [*ABORT] Return to SLY's top level.
 --more--

Backtrace:
 0: ("undefined function" #<SINGLE-MENU {1002953863}>)
 1: (RUN-MENU #S<screen #<XLIB:SCREEN :0.0 1920x1080x24 TRUE-COLOR>> #<SINGLE-MENU {1002953863}>)
 2: (GROUPLIST "%n%s%t")
 3: (SB-INT:SIMPLE-EVAL-IN-LEXENV (GROUPLIST) #<NULL-LEXENV>)
 4: (EVAL (GROUPLIST))
 5: ((LAMBDA NIL :IN SLYNK-MREPL::MREPL-EVAL-1))
 6: (SLYNK::CALL-WITH-RETRY-RESTART "Retry SLY mREPL evaluation request." #<FUNCTION (LAMBDA NIL :IN SLYNK-MREPL::MREPL-EVAL-1) {100295166B}>)
 7: ((LAMBDA NIL :IN SLYNK-MREPL::MREPL-EVAL-1))
 8: ((LAMBDA NIL :IN SLYNK::CALL-WITH-LISTENER))
 9: (SLYNK::CALL-WITH-BINDINGS ((*PACKAGE* . #1=#<PACKAGE "STUMPWM">) (*DEFAULT-PATHNAME-DEFAULTS* . #P"/home/katyusha/") (* . #1#) (**) (***) (/ #1#) ...) #<FUNCTION (LAMBDA NIL :IN SLYNK::CALL-WITH-LIST..
10: (SLYNK-MREPL::MREPL-EVAL-1 #<SLYNK-MREPL::MREPL mrepl-1-1> "(grouplist)")
11: (SLYNK-MREPL::MREPL-EVAL #<SLYNK-MREPL::MREPL mrepl-1-1> "(grouplist)")
12: (SLYNK:PROCESS-REQUESTS NIL)
13: ((LAMBDA NIL :IN SLYNK::SPAWN-CHANNEL-THREAD))
14: ((LAMBDA NIL :IN SLYNK::SPAWN-CHANNEL-THREAD))
15: (SLYNK-SBCL::CALL-WITH-BREAK-HOOK #<FUNCTION SLYNK:SLYNK-DEBUGGER-HOOK> #<FUNCTION (LAMBDA NIL :IN SLYNK::SPAWN-CHANNEL-THREAD) {100450000B}>)
16: ((FLET SLYNK-BACKEND:CALL-WITH-DEBUGGER-HOOK :IN "/home/katyusha/quicklisp/dists/quicklisp/software/sly-20210807-git/slynk/backend/sbcl.lisp") #<FUNCTION SLYNK:SLYNK-DEBUGGER-HOOK> #<FUNCTION (LAMBDA ..
17: ((LAMBDA NIL :IN SLYNK::CALL-WITH-LISTENER))
18: (SLYNK::CALL-WITH-BINDINGS ((*PACKAGE* . #1=#<PACKAGE "STUMPWM">) (*DEFAULT-PATHNAME-DEFAULTS* . #P"/home/katyusha/") (* . #1#) (**) (***) (/ #1#) ...) #<FUNCTION (LAMBDA NIL :IN SLYNK::CALL-WITH-LIST..
19: ((LAMBDA NIL :IN SLYNK::SPAWN-CHANNEL-THREAD))
20: ((FLET SB-UNIX::BODY :IN SB-THREAD::RUN))
21: ((FLET "WITHOUT-INTERRUPTS-BODY-10" :IN SB-THREAD::RUN))
22: ((FLET SB-UNIX::BODY :IN SB-THREAD::RUN))
23: ((FLET "WITHOUT-INTERRUPTS-BODY-3" :IN SB-THREAD::RUN))
24: (SB-THREAD::RUN)
25: ("foreign function: call_into_lisp")
```

**Comment by szos** [comment]
Im unable to reproduce this on the latest StumpWM master, is there any other information on how to trigger the error? I tried both through the `colon` command and via a (slynk) repl.

That being said I did find that after pressing return the message window was unmapped and then remapped, but only when text was typed into the search prompt, and i couldnt trigger it every time. I think this is due to the way help keys are displayed and how they manipulate the message window, but I'm unsure. It may be best to roll this back until a more comprehensive solution is found. Ill look into the help message when I have some time.
